components/common/accordion/accordion_item.tsx

```diff
@@ -75,9 +75,9 @@ const AccordionItem = ({
                         {data.extraContent}
                     </div>
                 )}
-                <div className='accordion-item-header__chevron'>
+                {data.items.length > 0 && <div className='accordion-item-header__chevron'>
```

Comment:
there is a early return above for `items.length === 0`, so this check is unnecessary.

Reply:
or is this in preparation for the "success" accordion item when there is nothing to do?

Reply:
or is this in preparation for the "success" accordion item when there is nothing to do?

> Yes! exactly, that way the item just looks like a card.